src/core/plugin/luapi_application.h

```diff
@@ -1217,6 +1243,11 @@ static int applib_changeBackgroundPdfPageNr(lua_State* L) {
  * Example 6: local highlighterInfo = app.getToolInfo("highlighter")
  *            local sizeName = highlighterInfo["size"]["name"]
  *            local opacity = highlighterInfo["fillOpacity"]
+ *
+ * Example 7: local selectionInfo = app.getToolInfo("selection")
+ *            local rotation = selectionInfo["rotation"]
+ *            local bounding_x = selectionInfo["boundingBox"]["x"]
+ *            local snappedBounds_width = selectionInfo["snappedBounds"]["width"]
```

Comment:
So far we used the naming conventions from the [wiki](https://github.com/xournalpp/xournalpp/wiki/Coding-conventions#naming-conventions) for Lua as well, so please stick with it.
```suggestion
 *            local boundingX = selectionInfo["boundingBox"]["x"]
 *            local snappedBoundsWidth = selectionInfo["snappedBounds"]["width"]
```